nutils/function.py

```diff
@@ -2664,10 +2664,10 @@ def get_support(self, dof: Union[numbers.Integral, numpy.ndarray]) -> numpy.ndar
       dof = numpy.unique(dof)
       if dof[0] < 0 or dof[-1] >= self.ndofs:
         raise IndexError('dof out of bounds')
-      if self.get_support == __class__.get_support.__get__(self, __class__):
-        return numpy.unique([ielem for ielem in range(self.nelems) if numpy.in1d(self.get_dofs(ielem), dof, assume_unique=True).any()])
-      else:
-        return numpy.unique(numpy.fromiter(itertools.chain.from_iterable(map(self.get_support, dof)), dtype=int))
```

Comment:
what was the reason for this distinction?

Reply:
The if branch is taken if there's no reimplementation of `getsupport`. I believe the reason was to minimize using the fallback `_computed_support`, but in retrospect I don't think it makes sense.